netflix-roulette/src/pages/home/components/Header/components/AddMovieButton/AddMovieButton.test.tsx

```diff
@@ -0,0 +1,48 @@
+import * as React from "react";
+import { render, fireEvent, RenderResult } from "@testing-library/react";
+import { AddMovieButton } from "./AddMovieButton";
+import { ThemeProvider } from "styled-components";
+import { theme } from "@utils/theme";
+import "@testing-library/jest-dom";
+import reactRedux from "react-redux";
+
+const renderComponent = (children: React.ReactElement): RenderResult =>
```

Comment:
I would say, relying on external dependencies (rather than mocking them) may seem to be a [test smell](https://testsmells.org/pages/testsmells.html#MysteryGuest).